sdk/messaging/azservicebus/sender_unit_test.go

```diff
@@ -45,14 +46,45 @@ func TestSender_UserFacingError(t *testing.T) {
 
 	var asSBError *Error
 
-	err = sender.SendMessage(context.Background(), &Message{}, nil)
+	msgID := "testID"
```

Comment:
We still want the original test (	err = sender.SendMessage(context.Background(), &Message{}, nil) here, but otherwise loving the addition of tracing to the mix.

Reply:
I think that might be git trying to trick us -- the test is still there on line 60!